.github/workflows/build.yml

```diff
@@ -192,18 +192,6 @@ jobs:
             echo "::error title=The Pre-Commit Checks Failed.::Please run 'pre-commit run --show-diff-on-failure'"
           fi
           exit "$RESULT"
-      - name: Check License Headers
```

Comment:
Included in invocation from https://github.com/gitpod-io/gitpod/pull/18239/files#diff-5c3fa597431eda03ac3339ae6bf7f05e1a50d6fc7333679ec38e21b337cb6721L186